hack/render-e2e-runner.sh

```diff
@@ -5,16 +5,29 @@ set -eu
 source hack/common.sh
 
 TMPFILE=$(mktemp /tmp/rune2e.XXXXXX)
-MARKER=$(grep -Fn '### serial suite runner' hack/run-test-serial-e2e.sh | cut -d: -f1)
+if [[ "$1" == 'must-gather' ]]; then
+    MARKER=$(grep -Fn '### must-gather suite runner' hack/run-test-must-gather-e2e.sh | cut -d: -f1)
+    TEST_FILE='hack/run-test-must-gather-e2e.sh'
+    FILE='run-e2e-nrop-must-gather.sh'
+    
+elif [[ $1 == 'serial' ]]; then
+    MARKER=$(grep -Fn '### serial suite runner' hack/run-test-serial-e2e.sh | cut -d: -f1)
+    TEST_FILE='hack/run-test-serial-e2e.sh'
+    FILE='run-e2e-nrop-serial.sh'
```

Comment:
this is good, and I wonder if we can untie this for only MG or serial tests. I agree we need to have a default script, and serial it is, but what if we want to execute other tests later on that are containerized in the image, but because of this limitation we can't actually reach and run them freely.